doc/memos/rdm0.md

```diff
@@ -0,0 +1,128 @@
+about architectural design decisions, code structure etc. Other memos might also
+describe other aspects of RIOT activity, including, but not limited to, RIOT 
+community processes, position with respect to some related external technical 
+debates etc.
+
+
+This memo specifies a format for such memos and the process of discussing their 
+content before publication. Format and processes are somewhat inspired by 
+[IETF](http://ietf.org/) processes and documents such as Request For Comments 
+(RFC).
+
+# 2. RIOT Developers Memo Format
+
+RIOT Developer Memos (RDM) are written in English, using the Markdown (.md) 
+format, with less than 80 character per line. Each memo has a unique identification number indicated in its header, 
+e.g. this memo is RDM0.
+
+The structure of a memo should contain:
+
+1. a preamble as defined below,
+2. the content of the memo structured as needed,
+3. a postamble as defined below.
+
+The preamble of the memo should be structured as such:
+
+1. a header formated as the header of this memo,
+2. a short abstract,
+3. a status for the memo stating its license and the category of the memo (e.g. 
+that it is a consensus document),
+4. optionally, a terminology section which recalls/disambiguates definition of 
+terms that are key to the memo.
+
+The core content of the memo is structured as deemed necessary. However, during 
+the review process, care must be taken to follow the below principles as much as
+possible:
+
+1. one memo = one problem/aspect,
+2. start by crisply explaining and motivating the one problem/aspect the memo 
+targets,
+3. memos are expected to be of normative and durable nature, hence: keep a memo 
+as concise as possible without impairing clarity, and factor out content that 
+would be likely to be updated too often.
+
+The postamble of the memo should be structured as such:
+
+1. (optional) acknowledgements section thanking contributors other than the 
+authors,
+2. a section with the main references that relate to the content, with a strong
+preference for permanent, self-contained references,
+3. contact information to reach the author(s).
+
+
+# 3. Memo Publishing and Maintenance Process
+
+Each memo is proposed as a pull-request (PR) and discussed the same way RIOT
+code PRs are processed on GitHub.
+
+
+Once proposed, a memo must have received an ACK by at least 3 maintainers other 
+than the author(s) of the memo, before its publication. 
+
+
+At this stage, the RDM is assigned the next available RDM number, and merged in the master branch of RIOT. Once merged into the master branch of RIOT, a memo is considered published.
+
+Small changes/clarifications may be subsequently PRed to improve a memo, using the same RDM number. Bigger changes should result in a new memo with another RDM number, deprecating the old memo. In this case, the deprecated memo must be modified to append a line in its header indicating "DEPRECATED BY RDM X" where X is the RDM number of the new memo.
```

Comment:
OK. How about now?However, I'm not in favor of adding the date, it feels like potential chicken-egg issue at time of merging, like "please ACK" one day goes by "please change date" REPEAT forever. And the dates are in the commit history ;)